### ISSUE TITLE
Parameters for model Training

### ISSUE THREAD

**Issue by prkbuilds** [polo8214/Brain-tumor-segmentation#13]
`#history = model.fit(x, y, batch_size=16, validation_split=0,validation_data = (val_x,val_y) ,epochs = 40,callbacks = callbacks_list ,verbose=1, shuffle=True)`
I have the dataset with me, i'm trying to train the model but can't get what are the x and y in the above line. Also it would be great if you tell how you have defined x and y. 
Please Help
